app/src/main/java/org/oppia/android/app/walkthrough/WalkThroughVerticalScrollListener.kt

```diff
@@ -0,0 +1,10 @@
+package org.oppia.android.app.walkthrough
+
+/** Listener for when an activity should hide and show ProgressBar And HeaderTextView. */
+interface WalkThroughVerticalScrollListener {
+  /** Hides ProgressBar and show HeaderTextView*/
```

Comment:
Nit: Add space at the end `/** Hides ProgressBar and show HeaderTextView */`

Reply:
Done